client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPluginValidator.js

```diff
@@ -303,3 +307,7 @@ function hash(el) {
 function shallowEquals(a, b) {
   return hash(a) === hash(b);
 }
+
+function validCloudUrl(url) {
+  return /^(https:\/\/|)[a-z\d-]+\.[a-z]+-\d+\.zeebe\.camunda\.io:443/.test(url);
```

Comment:
Port 443 is always used on Camunda Cloud so I don't see a need to provide it explicitly.

```suggestion
  return /^(https:\/\/|)[a-z\d-]+\.[a-z]+-\d+\.zeebe\.camunda\.io/.test(url);
```

Reply:
I've tested the suggested rule and it works correctly.

Reply:
On the other hand, we should not accept other ports...

Reply:
Yep, menski confirmed it would always be 443 for Camunda Cloud. So I propose we include it in the Regex, since the deploy will otherwise fail anyhow

Reply:
I think I have a solution:

```suggestion
  return /^(https:\/\/|)[a-z\d-]+\.[a-z]+-\d+\.zeebe\.camunda\.io(:443)?\/?$/.test(url);
```

So it should be either :443 or nothing. A trailing slash should also work as it does not change the URL.

Reply:
What I'd like to avoid is a situation when somebody pastes an https link (where port 443 is the default) and receives an error message.

Reply:
Ah - sure, did not think of that case. Thanks. Will adjust

Reply:
(BTW: we extract the clusterID and region from the provided URL (see https://github.com/camunda/camunda-modeler/pull/2390/commits/c3e918377d5208221f96e0924b1855d905f469e0) since zeebe-node does not accepts full URLs since 1.4.0. So this is not super important for us to make the deploy work, still I think it is good practice to ensure proper input by the user.)

Reply:
true, this could be anything 🦀

Reply:
https://github.com/camunda/camunda-modeler/pull/2390/commits/9a332164dcd8b140887fb954a5dace6d2593dbe5